tests/unit/BusinessContactInfo.spec.ts

```diff
@@ -42,135 +43,180 @@ function getLastEvent (wrapper: Wrapper<BusinessContactInfo>, name: string): any
   return events[0]
 }
 
+/**
+ * Utility method to get around with the timing issues
+ */
+async function waitForUpdate (wrapper: Wrapper<Vue>) {
+  await Vue.nextTick()
+  await flushPromises()
+  await Vue.nextTick()
```

Comment:
I like this method! Though i am curious why the double `Vue.nextTick()`

Reply:
Sometimes even though the tests pass in local, they fail when run as part of github actions. After trial and error we found that this one works consistently. But I dont know why :)

Reply:
haha totally strange but i understand completely! I literally had this exact issue in my recent PR, i had to use flush promises over nextTick() to get them to pass in gha as well. 

Sounds good!